packages/docs/src/theme/Navbar/Content/styles.module.css

```
@@ -0,0 +1,14 @@
+/* Swizzled from @docusaurus/theme-classic Navbar/Content (v3.8.1). */
+
+/* Hide color mode toggle in small viewports */
+@media (max-width: 996px) {
+	.colorModeToggle {
+		display: none;
+	}
+}
+
+/* Restore some Infima style that broke with CSS Cascade Layers.
+   See https://github.com/facebook/docusaurus/pull/11142 */
+:global(.navbar__items--right) > :last-child {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)